pmpsdb_client/pmpsdb_lfe.yml

```diff
@@ -1,3 +1,5 @@
 plc-lfe-motion: "PLC:LFE:MOTION"
 plc-lfe-optics: "PLC:LFE:OPTICS"
+plc-lfe-vac: "PLC:LFE:VAC"
```

Comment:
@ghalym do you remember why we didn't include the KFE vacuum PLC when setting up the K line? Does the LFE vacuum PLC need any pmps database support? What's our actual scope here?

Reply:
I chatted with Maggie and we concluded that plcs named `-vac` and also the lfe GEM (`plc-lfe-gatt`) did not need to be (should not be) included in the database.